racksdb/web/ui/src/views/InfrastructureDetailsView.vue

```diff
@@ -48,15 +48,6 @@ onMounted(() => {
   getInfrastructureImg()
 })
 
-// Using watch to trigger getDatacenter() when the value of props.name change
```

Comment:
This change does not seem linked to this feature, is it?

Reply:
Well yes and no, this code was creating this error:
_Uncaught (in promise) ReferenceError: can't access lexical declaration 'props' before initialization_

I no longer had access to all the infrastructure details, I don't know why this error didn't appear earlier, maybe it's since I updated Vue.